ruff.toml

```diff
@@ -1,4 +1,4 @@
-tab-size = 2
+indent-width = 2
```

Comment:
Why did this change?

Reply:
ruff emits a warning when `tab-size` is used, which suggests this change (see [here](https://docs.astral.sh/ruff/settings/#tab-size)). The effect is the same, but I can revert if desired.

Reply:
nah it's fine